lib/src/json_schema/json_schema.dart

```diff
@@ -907,6 +926,7 @@ class JsonSchema {
     'definitions': (JsonSchema s) => s.definitions,
     'properties': (JsonSchema s) => s.properties,
     'items': (JsonSchema s) => s.items ?? s.itemsList,
+    'prefixItems': (JsonSchema s) => s.prefixItems,
```

Comment:
I build this really complex thing so references could point to items in `prefixItems` lists! Then I found this data structure and replace it all with this 1 line.